Formula/qt.rb

```diff
@@ -61,31 +34,13 @@ def install
       -qt-libjpeg
       -qt-freetype
       -qt-pcre
+      -nomake examples
       -nomake tests
       -no-rpath
       -pkg-config
       -dbus-runtime
     ]
 
-    args << "-nomake" << "examples" if build.without? "examples"
-
-    if build.with? "mysql-client"
-      args << "-plugin-sql-mysql"
-      (buildpath/"brew_shim/mysql_config").write <<~EOS
-        #!/bin/sh
-        if [ x"$1" = x"--libs" ]; then
-          mysql_config --libs | sed "s/-lssl -lcrypto//"
-        else
-          exec mysql_config "$@"
-        fi
-      EOS
-      chmod 0755, "brew_shim/mysql_config"
-      args << "-mysql_config" << buildpath/"brew_shim/mysql_config"
-    end
-
-    args << "-plugin-sql-psql" if build.with? "postgresql"
-    args << "-proprietary-codecs" if build.with? "proprietary-codecs"
```

Comment:
I think you should add back `-proprietary-codecs` which was default behaviour before (the option was `--without-`).

Reply:
thanks for catching that! I've added it back and squashed in 8c2e33f